prboom2/src/m_cheat.c

```diff
@@ -999,6 +999,21 @@ dboolean M_CheatResponder(event_t *ev)
   return false;
 }
 
+dboolean M_CheatEntered(char element[], char value[3])
+{
+  cheatseq_t* cheat_i;
+
+  for (cheat_i = cheat; cheat_i->cheat; cheat_i++)
+  {
+    if (!strcmp(cheat_i->cheat, element) && M_CheatAllowed(cheat_i->when))
```

Comment:
So this will work for everything _except_ idclev, is that correct? Any idea why this cheat is disabled in the menu? Maybe we can address that later on.

Reply:
I removed the "not_menu" flag and everything worked correctly, but when using idclev on the console it would warp to the level, but not update the display until the console was exited.
So this cheat was probably disabled to avoid this visual problem.

Reply:
> So this will work for everything _except_ idclev, is that correct? Any idea why this cheat is disabled in the menu? Maybe we can address that later on.

Check the 5th bullet point:
https://doomwiki.org/wiki/IDCLEV_idiosyncrasies